.buildkite/pipeline.yml

```diff
@@ -68,4 +68,12 @@ steps:
     depends_on: "java_integration_tests"
     command:
       - 'buildkite-agent artifact download "build/*" .'
+      - "env"
       - "./.buildkite/scripts/steps/upload_to_s3.sh"
+    agents:
+      cpu: "16"
+      ephemeralStorage: "20G"
+      memory: "32G"
```

Comment:
This step shouldn't need anything like 16 CPUs or 32GB of RAM to work.

I would think 2 CPUs and 4GB of RAM would be fine.  Or if not, why is uploading a file so memory hungry?